mlflow/pytorch/__init__.py

```diff
@@ -725,6 +726,102 @@ def predict(self, data, device="cpu"):
             return predicted
 
 
+@experimental
+def log_state_dict(state_dict, artifact_path, **kwargs):
+    """
+    Log a state_dict as an MLflow artifact for the current run.
+
+    :param state_dict: state_dict to be saved.
+    :param artifact_path: Run-relative artifact path.
+    :param kwargs: kwargs to pass to ``torch.save``.
+
```

Comment:
```suggestion
.. warning:
    We currently don't support logging a state_dict as an MLflow model.
```

@smurching Shoule we mention that we don't supprt logging a state_dict as an MLflow model to avoid confusing users?